documentation/API_documentation/Connectivity_API.md

```diff
@@ -0,0 +1,82 @@
+# Overview
+This API makes possible to query the connectivity status of specific user equipment.
+
+## 1\. Introduction
+<img src="resources/DeviceStatus_Connectivity_diagram.png">
+
+## 2\. Quick Start
+
```

Comment:
The usual basics needed for quickstart cud be copied over from qod docu

Reply:
Also the API version should be included, in the location API doc as well